src/targets/quadkopters_2400.ini

```diff
@@ -0,0 +1,31 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:QuadKoters_JR_2400_TX_via_UART]
```

Comment:
should this not be QuadKopters not QuadKoters?

Reply:
Yea thanks for pointing this out *embarrassed*